assets/scss/hero.scss

```diff
@@ -29,61 +53,85 @@
   }
 
   &.browse-jumbotron {
-    .container {
-      padding-bottom: 3rem;
+    margin-top: 2rem;
+    border-radius: 0.5rem;
+    overflow: hidden;
+    &:before {
+      display: none;
```

Comment:
Is there a before?

Reply:
yes, the before is still used on jumbotron images on the exhibition, it adds a semi-transparent dark cover over images